### ISSUE TITLE
Suggestion: use service icon in notifications

### ISSUE THREAD

**Issue by ThHareau** [sonnyp/Tangram#221]
Hi! 

I'm discovering the service, it's very nice! 

Would it be possible for desktop notifications to use the service icon instead of Tangram's? it would be clearer

**Comment by luchmhor** [comment]
Yes, Desktop notifications / a tray icon would be very beneficial for this applications, especially when using it for a set of messaging tabs.
